twinPicks/jsonld-contexts/twinPicks-compound.jsonld

```diff
@@ -1,11 +1,17 @@
 {
   "@context": [
-    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat-compound.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/datacenter-equipments.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/dcat/jsonld-contexts/dcat.jsonld",
+    "https://easy-global-market.github.io/c2jn-data-models/jsonld-contexts/c2jn-smart-city-compound.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/archive/digital-twins/jsonld-contexts/dcat.jsonld",
     "https://easy-global-market.github.io/ngsild-api-data-models/twinPicks/jsonld-contexts/usecase.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/device/jsonld-contexts/device.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/environment/jsonld-contexts/environment.jsonld",
+    "https://easy-global-market.github.io/ngsild-api-data-models/weather/jsonld-contexts/weather.jsonld",
```

Comment:
don't think they should go in the core TP context

Reply:
I'm not sure about those... There is a big chance that they will be needed in all usecases.

Reply:
Removed